HTML/img/Component/Baybayin Words.svg

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You'll need to put this file through [SVG minification](https://jakearchibald.github.io/svgomg/).